platforms/core-runtime/files/src/main/java/org/gradle/internal/file/nio/ModificationTimeFileAccessTimeJournal.java

```diff
@@ -25,6 +25,7 @@
 import java.nio.file.Files;
 import java.nio.file.attribute.FileTime;
 
+@SuppressWarnings("Since15")
```

Comment:
❓ Is this safe to suppress? This suppression means that we use Java features that need some higher JDK, right?

If this is safe should we add a comment why this is safe.

Reply:
FTR, we discussed this face-to-face that these new warnings are due to the fact that the classes were moved out of other subprojects where this check was suppressed. The pattern of mixing Java 8 code in Java 6-bound subprojects occurs in several other places, too, and we should address separately.